pages/docs/configuration/configuration-communication.md

```diff
@@ -15,21 +15,21 @@ For each two participants that should exchange data, you have to define an m2n c
 
 This establishes an `m2n` (i.e. parallel, from the M processes of the one participant to the N processes of the other) communication channel based on TCP/IP `sockets` between `MySolver1` and `MySolver2`.
 
-For certain systems, you need to specify the network over which the TCP/IP sockets get connected: `network="..."`. It defaults to `"lo"`. For some clusters, you could use the infiniband, e.g. `"ib0"`. macOS is also a [special case](macOS).  
```

Comment:
This GitHub Action does not check such local links. It also did not catch the `TODO` (but I searched and fixed all of the `TODO` ones).

Reply:
See open feature request: https://github.com/tcort/markdown-link-check/issues/91